### PR TITLE
Add Alert Component

### PR DESCRIPTION
**Pull request by varun13169**
Add Alert Component
	- alert html and css file.
	- assets required for the component

### CODE REVIEW

**Comment by varun13169** [comment]
Preview
image

**Review by zoef-yt — APPROVED** [review]
This looks good.

File: components/alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alert Component</title>
+    <link rel="stylesheet" href="/library/component.css">
+    <style>
+        body {
+            padding: 0px;
+        }
```
Comment (sanu29) [inline components/alert/alert.html:12]: Internal CSS could be avoided you could make another file for this

**Review by sanu29 — APPROVED** [review]
Looks Great for me. Quick suggestions : 
You can remove the extra spaces from CSS file.
File structure could be better like all the svg could be in one folder 
But yet all good.. great work

File: components/alert/alert.html (hunk 1 above)
Comment (varun13169) [reply components/alert/alert.html:12]: Hi @sanu29,
Following code is part of the support code for the CSS component library.
Will be refactoring it once **library P0s are complete.** and structure is hardened.

**Comment by varun13169** [comment]
> This looks good.

Thanks @zoef-yt please Approve if things look good :)

File: library/alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+.alert__text--error {
+    color: var(--alert-error-light-font-color);
+}
+/* Alert Component Ends -- Error */
+
+
+/* Alert Component Starts -- Warning */
+.alert--warning {
+    background: rgba(var(--rgba-alert-warning-light-background-color));
```
Comment (iRohitGaur) [inline library/alert/alert.css:46]: why rgba here?

Comment (varun13169) [reply library/alert/alert.css:46]: To provide alpha value, Just checked and realized we can use alpha values in hex as well. 
Will be refactoring the code accordingly.

Thanks for pointing it out.

**Pull request merged**